crates/primitives/src/sealed.rs

```diff
@@ -5,6 +5,7 @@ use crate::B256;
 /// We do not implement any specific hashing algorithm here. Instead types
 /// implement the [`Sealable`] trait to provide define their own hash.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "rlp", derive(alloy_rlp::RlpEncodable, alloy_rlp::RlpDecodable))]
```

Comment:
I don't think this is useful because the hash is usually never included in any rlp payload if it can be computed from the sealed element

Reply:
actually, I can see this being useful for decodeable where T: Sealable